includes/class-sensei-course.php

```diff
@@ -1155,50 +1155,13 @@ public function course_lesson_preview_count( $course_id = 0 ) {
 	 */
 	public function get_product_courses( $product_id = 0 ) {
 
-		$courses = array();
+		_deprecated_function( __CLASS__ .':'. __METHOD__, '2.0.0', 'Sensei_WC_Paid_Courses\Courses::get_product_courses' );
```

Comment:
It seems like only __METHOD__ is needed here as this currently outputs the following to the logs:

```
PHP Notice:  Sensei_Course:Sensei_Course::get_product_courses is <strong>deprecated</strong> since version 2.0.0! Use Sensei_WC_Paid_Courses\Courses::get_product_courses instead. in /app/public/wp-includes/functions.php on line 3923
```